docs/data/toolpad/building-ui/datagrid-component.md

```diff
@@ -0,0 +1,38 @@
+# DataGrid component
+
+<p class="description">Build powerful and feature rich data tables</p>
+
+<img src="/static/toolpad/docs/building-ui/datagrid.png" width="884px" alt="Custom component" />
+
+## Customizing cell rendering
+
+By default you can render json data using DataGrid component and it will automatically recognize most of the data types. However sometimes it might not be enough and for that reason we provide the option to render custom components inside a datagrid.
```

Comment:
The content seems fine to me! But for improving the grammar and punctuation we could run this by Sam maybe?

Reply:
```suggestion
By default, you can render JSON data using the Data Grid component and it will automatically recognize most of the data types. 
If this isn't sufficient, you can also render custom components inside the Data Grid.
```